torch/csrc/jit/serialization/flatbuffer_serializer.cpp

```diff
@@ -320,9 +321,15 @@ flatbuffers::DetachedBuffer FlatbufferSerializer::serializeModule(
     storage_data_offset = fbb.CreateVector(storage_data);
   }
 
+#if ENABLE_UPGRADERS
+  const uint32_t version = caffe2::serialize::kMinProducedFileFormatVersion;
+#else
+  const uint32_t version = caffe2::serialize::kProducedFileFormatVersion;
```

Comment:
is the version always be either of these two no matter what operator the model has?

Reply:
Q: What is the correct way to find operator version?